#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

PACKAGE = "r1_laser_filter"

gen = ParameterGenerator()

gen.add("max_range", double_t, 0, "Only ranges smaller than this range are taken into account", 2.0, 0, 100)
gen.add("max_range_difference", double_t, 0, "Distance: max distance between consecutive points - RadiusOutlier: max distance between points", 0.4, 0, 100)
gen.add("filter_window", int_t, 0, "Minimum number of neighbors", 4, 0, 100)

exit(gen.generate(PACKAGE, "r1_laser_filter", "SpeckleFilter"))
